src/extra/gfx/material_utils.hpp

```diff
@@ -11,6 +12,25 @@
 #include <spdlog/spdlog.h>
 
 namespace gfx {
+inline void varToTexture(const SHVar &var, TexturePtr &outVariant) {
+  switch (var.valueType) {
+  case SHType::Object: {
+    shards::Type type =
+        SHTypeInfo{SHType::Object, {.object = {.vendorId = var.payload.objectVendorId, .typeId = var.payload.objectTypeId}}};

```

Comment:
Opportunity to use `Type::Object(var.payload.objectVendorId, var.payload.objectTypeId)` once #284 is merged.